bin/install-chromedriver

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+gem install webdrivers
+ruby -e "require 'webdrivers/chromedriver'; Webdrivers::Chromedriver.update; puts 'Chromedriver installed to' + Webdrivers.install_dir"
```

Comment:
Maybe this should spit out the `/chromedriver` part at the end, so it can just be straight copy-pasted into what you set for CHROMEDRIVER_PATH?